lib/engine/game/base.rb

```diff
@@ -1073,6 +1073,13 @@ def sell_shares_and_change_price(bundle, allow_president_change: true, swap: nil
           stock_market.move_down(corporation) if was_president
         when :left_block_pres
           stock_market.move_left(corporation) if was_president
+        when :left_per_if_pres_else_left_one
```

Comment:
what does the naming mean? left_per?

Reply:
I agree this is confusing.  left_per_10_if_pres_else_left_one is clearer.

Reply:
Changed